src/style.rs

```diff
@@ -176,3 +235,28 @@ impl FromStr for Margin {
         }
     }
 }
+
+impl FromStr for Radius {
+    type Err = &'static str;
+
+    fn from_str(s: &str) -> Result<Self, Self::Err> {
+        let values = s
+            .split(' ')
+            .map(|s| s.trim())
+            .filter(|s| !s.is_empty())
+            .map(|s| s.parse::<f32>().map_err(|_| "invalid float value"))
```

Comment:
```suggestion
            .map(|s| s.parse::<f32>().map_err(|_| format!("invalid float value: {:?}", s)))
```

so no need to guess which one is incorrect

Reply:
👍🏾